lib/decorators.js

```diff
@@ -122,7 +122,6 @@ class History extends Decorator {}
 // Not materialized entity decorators
 
 class View extends Decorator {}
-class Projection extends Decorator {}
```

Comment:
Is the `Projection` decorator removal relevant to the modules support?

Reply:
We would need to add support for `.projection` file extension, and remove it in a followup PR.